tests/reference_data/water_rotations.xyz

```diff
@@ -0,0 +1,10 @@
+3
+Properties=species:S:1:pos:R:3 #=T Bead:=T 100.00000=T 22860=T positionsangstrom=T CELLabcABC:=T 0=T 90.00000=T pbc="F F F" Step:=T cellatomic_unit=T
```

Comment:
Is this the output of ASE? Can that be read in again?

Reply:
this is the libatoms extended xyz format, and yes.

Reply:
Apparently not, because when ASE reads it, the cell is discarded. Looks like the input or output of some nonstandard tool.